scripts/linux-aarch64/update.sh

```diff
@@ -57,6 +58,22 @@ cp -r data albyhub-backup
 
 echo "Downloading latest version"
 wget $ALBYHUB_URL
+
+if [[ ! -f "verify.sh" ]]; then
+  echo "Downloading the verification script..."
+  if ! wget -q "$VERIFIER_URL"; then
+    echo "❌ Failed to download the verification script." >&2
+    exit 1
+  fi
+  chmod +x verify.sh
+fi
+
+./verify.sh server-linux-aarch64.tar.bz2 albyhub-Server-Linux-aarch64.tar.bz2
+if [[ $? -ne 0 ]]; then
+  echo "❌ Verification failed, aborting installation"
```

Comment:
What happens if the user hasn't imported the gpg key yet?

Should we update this message to also include: `. Visit https://github.com/getAlby/hub/releases for more information on how to verify the release`

Reply:
Edit: moved this question.